.travis.yml

```diff
@@ -28,7 +28,7 @@ before_install:
   - cd; cd lammps/lib/latte; ln -s $LATTEPATH/src/latte_c_bind.o filelink.o ; ln -s $LATTEPATH/src includelink ; ln -s $LATTEPATH liblink
   - cd; cd lammps/lib/latte; cp Makefile.lammps.gfortran Makefile.lammps
   - cd; cd lammps/lib/latte; echo "latte_SYSLIB  += -fprofile-arcs -ftest-coverage -O0" >> Makefile.lammps  
-  - cd; cd lammps/src/; make yes-latte; make yes-molecule
+  - cd; cd lammps/src/; if [[ ${LATTE_CMAKE} != yes ]]; then make yes-latte; make yes-molecule; fi
```

Comment:
This is needed as the recent lammps is more picky about not mixing make and cmake buildsystems.